lib/apiForSheets.js

```diff
@@ -95,19 +96,31 @@ module.exports.generateOutput = async function (rows, prNumber, prDetails) {
       'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#' +
       'setting-things-up'
     );
+    var linkOppiaAndroid = 'onboarding instructions'.link(
+      'https://github.com/oppia/oppia-android/wiki#onboarding-instructions'
+    );
+
+    var commentBody = '';
+    if (repositoryName === 'oppia-android') {
+      commentBody = 'Hi! ' + userNamesNotSignedCLA +
+        'Welcome to Oppia! Please could you follow the ' +
+        linkOppiaAndroid + ' to get started with oppia-android? ' +
+        'You\'ll need to do this before we can accept your PR. I am ' +
+        'closing this PR for now. Feel free to re-open it once you ' +
+        'are done with the above instructions. Thanks!'
```

Comment:
The comment generated here is different from that being asserted in your test. PTAL.

Reply:
working on requested changes.

Reply:
addressed all changes in the latest commit.